src/core/js/models/adaptModel.js

```diff
@@ -218,6 +218,60 @@ define([
             return returnedDescendants;
         },
 
+        
+        getAllDescendantModels: function(isParentFirst) {
```

Comment:
sorry to be picky but could you move these style of comments to be above the function definition instead of inside the function body?